src/win/winapi.c

```diff
@@ -109,4 +114,12 @@ void uv_winapi_init() {
     pPowerRegisterSuspendResumeNotification = (sPowerRegisterSuspendResumeNotification)
       GetProcAddress(powrprof_module, "PowerRegisterSuspendResumeNotification");
   }
+
+  secur32_module = LoadLibraryA("secur32.dll");
```

Comment:
this API (GetUserNameEx) is supported from Windows 2000, can't we just link with secur32 instead of dynamically loading it? We used to do dynamic loading because some APIs were not supported in newer Windows versions, but this one is, so we should be able to just call it.